dan_layer/consensus/src/hotstuff/on_receive_requested_transactions.rs

```diff
@@ -0,0 +1,49 @@
+//   Copyright 2023 The Tari Project
+//   SPDX-License-Identifier: BSD-3-Clause
+
+use log::*;
+use tari_dan_storage::consensus_models::ExecutedTransaction;
+use tokio::sync::mpsc;
+
+use crate::{
+    hotstuff::error::HotStuffError,
+    messages::RequestedTransactionMessage,
+    traits::{ConsensusSpec, EpochManager},
+};
+
+const LOG_TARGET: &str = "tari::dan::consensus::hotstuff::on_receive_requested_transactions";
+
+pub struct OnReceiveRequestedTransactions<TConsensusSpec: ConsensusSpec> {
+    tx_new_transaction: mpsc::Sender<ExecutedTransaction>,
+    _phantom: std::marker::PhantomData<TConsensusSpec>,
+}
```

Comment:
This generic is unused
```suggestion
pub struct OnReceiveRequestedTransactions {
    tx_new_transaction: mpsc::Sender<ExecutedTransaction>,
}
```

Reply:
The `TConsensusSpec` is needed for the `from` field in handle.